apps/cli/src/cli/web.ts

```diff
@@ -60,6 +64,8 @@ function makeDeployCLI(program: Command) {
     .addOption(makeForceOption())
     .addOption(makeYesOption())
     .addOption(makeCIOption())
+    .addOption(makeStrictOption())
```

Comment:
Should these have `true` defaults here?

Reply:
I don't think so, we want the build/deploy to be permissive & fast by default.